src/main/resources/com/google/api/codegen/go/doc.snip

```diff
@@ -10,6 +10,15 @@
     @join line : view.packageDoc
         // {@line}
     @end
+    //
+    // Use of Context
+    //
+    // The ctx passed to NewClient is used for authentication requests and
+    // for the underlying connection, which is shared between requests.
```

Comment:
// The ctx passed to NewClient is used for authentication requests and
    // for the underlying connection, which is shared between requests.

Wait - we need to be careful here. The context is passed to gRPC is _only used for setup_. https://godoc.org/google.golang.org/grpc#DialContext

We should not describe it as being used for the whole connection and all subsequent requests.

However, this might be the behavior for http. On the bus so bit unable to dig. cc @broady

Reply:
good catch. yes, all gapics are gRPC.

so yes, we should say it's only used for NewClient and they need to close the client if they want to shut connections.

Reply:
PTAL

Reply:
"methods -> "subsequent calls" or "API calls" I think?
"methods" is a gRPC concept (and a poorly defined Go concept)

otherwise LGTM

Reply:
+1 chris' comments, otherwise LGTM.

Reply:
Done. I will merge once CI finishes.